table/block_based/block_based_table_reader.h

```diff
@@ -233,11 +233,14 @@ class BlockBasedTable : public TableReader {
 
  protected:
   Rep* rep_;
-  explicit BlockBasedTable(Rep* rep) : rep_(rep) {}
+  explicit BlockBasedTable(
+      Rep* rep, AtomicBlockCacheTraceWriter* const block_cache_tracer = nullptr)
```

Comment:
Removing `= nullptr` causes PR to grow much bigger? If not, I prefer to remove it.

Reply:
I removed the default value.